task04/task04/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task04
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var x = GetValue();
+            Console.WriteLine("x = " + x);
+            Console.ReadKey();
+        }
+        
+        static double GetValue()
+        {
+            return F(1, 2, 3) + F(5, 3, 8) + F(1, 5, 6);
+        }
+        
+        static double F(double a, double b, double c)
+        {
+            return Math.Sqrt((a + Math.Pow(Math.Tan(b), 2)) / c);
```

Comment:
Ваше решение работает, однако можно было сделать еще проще
```suggestion
        static double GetValue()
        {
            return F(1, 2) + F(5, 3) + F(1, 5);
        }
        
        static double F(double a, double b)
        {
            return Math.Sqrt((a + Math.Pow(Math.Tan(b), 2)) / (a + b));
```